hyp3_sdk/jobs.py

```diff
@@ -14,6 +14,9 @@
 # TODO: actually looks like a good candidate for a dataclass (python 3.7+)
 #       https://docs.python.org/3/library/dataclasses.html
 class Job:
+    _finished_job_keys = {'job_id', 'logs',  'request_time', 'status_code', 'user_id'}
+    _successful_job_keys = _finished_job_keys.union({'browse_images', 'expiration_time', 'files', 'thumbnail_images'})
```

Comment:
Somewhat out of scope, but I notice we don't document a Job's member attributes anywhere. Is there a docstring-ish way to provide a sentence describing each?

Reply:
```py
class ExampleError(Exception):
    """Exceptions are documented in the same way as classes.

    The __init__ method may be documented in either the class level
    docstring, or as a docstring on the __init__ method itself.

    Either form is acceptable, but the two should not be mixed. Choose one
    convention to document the __init__ method and be consistent with it.

    Note:
        Do not include the `self` parameter in the ``Args`` section.

    Args:
        msg (str): Human readable string describing the exception.
        code (:obj:`int`, optional): Error code.

    Attributes:
        msg (str): Human readable string describing the exception.
        code (int): Exception error code.

    """

    def __init__(self, msg, code):
        self.msg = msg
        self.code = code
```

Reply:
Agreed, we should do this! I started in and I think I'd like to do this in a seperate PR. Recorded this in #130